src/main/java/core/basesyntax/model/User.java

```diff
@@ -8,6 +8,12 @@ public class User {
     private String password;
     private Integer age;
 
+    public User(String login, String password, Integer age) {
+        this.login = login;
+        this.password = password;
+        this.age = age;
+    }
```

Comment:
you don't need to change user file in this task

Reply:
this is fine, we can add constructors for a handy way of creating users in tests